content/applications/marketing/sms_marketing/essentials/sms_essentials.rst

```diff
@@ -176,6 +176,59 @@ is visualized as different graphs and charts, which can be sorted and grouped in
    :align: center
    :alt: Reporting page in SMS Marketing.
 
+.. tip::
+   SMS messages can be sent using automated rules in Odoo. Odoo *Studio* is required to use
+   automated rules. To install it, go to :menuselection:`Apps --> Search...` bar and search for
+   `studio`. If it is not already installed, click :guilabel:`Install`. Adding the *Studio*
+   application upgrades the subscription status to *Custom*, which increases the cost. Consult
+   `support <https://www.odoo.com/contactus>`_ or reach out to the database's customers success
+   manager with any questions on upgrading.
+
+   To use automated rules, navigate to :menuselection:`Settings app --> Technical menu -->
```

Comment:
don't forget to mention that users must be in developer mode in order to reveal the Technical Menu...

Reply:
@ksc-odoo 

This isn't true for Odoo 17.
![2024-04-24_14-12](https://github.com/odoo/documentation/assets/91896312/7aa9ae86-28ac-4053-9d72-ab8c1bb34700)

Reply:
![Screenshot 2024-04-24 111937](https://github.com/odoo/documentation/assets/73958186/aeb4fb75-7b8d-4e42-bf44-046de4a7f6da)

Reply:
That's wild! 

@ksc-odoo 

So I replicated both ways (with and without dev mode turned on) for a trial db - (non edu) in 17.0 

I think we go with activating developer mode (to cover all bases)

Thanks for troubleshooting this with me! :+1: 

Tim